startDevelopmentStack.sh

```diff
@@ -16,7 +16,15 @@ source ./*github_env.log
 source ./environments/.env.uncritical
 set +o allexport
 
-find ./build-utils/ -name "rebuild*.sh" ! -name "*prod*" ! -name "*test*" ! -name "*backend*" -exec bash -c 'eval "$1" && echo "SUCCESS - execution of $1 was successful" || echo "ERROR - could not execute $1"' shell {} \;
+if [ $# -ge 1 ] && [ "$1" == "parallel" ]; then
+  echo "Building docker images in parallel for unrivaled speed! (Does not work in Windows Git Bash)"
```

Comment:
Not easily. It uses GNU Parallel to run the docker build scripts in parallel (which is a standard linux utility). This utility is neither included in the git bash nor in the powershell. Nevertheless, it may potentially be possible to install GNU Parallel in git bash (But unfortunately not easily). But it does speed things up for me :)